firefox-ios/firefox-ios-tests/Tests/ClientTests/XCTestCaseExtensions.swift

```diff
@@ -64,4 +64,35 @@ extension XCTestCase {
         TelemetryWrapper.hasTelemetryOverride = false
         DependencyHelperMock().reset()
     }
+
+    // MARK: Error Handling
```

Comment:
This is where the convenience methods lie, I will make 2 commits:
- one with verify 
- one without verify